### PR TITLE
Readmeupdate09092020 deux

### PR DESCRIPTION
**Pull request by htharker42**
### What issue is this solving?
Update Readme with database info 

### Any helpful knowledge/context for the reviewer?
Is a re-seeding of the database necessary? no
Any new dependencies to install? no
Any special requirements to test? no

### Please make sure you've attempted to meet the following coding standards
- [ X] Code builds successfully
- [X ] Code has been tested and does not produce errors
- [X ] Code is readable and formatted
- [X ] There isn't any unnecessary commented-out code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +69,13 @@ If you are still having issues or if your command prompt is throwing an error th
 6. Run: `pipenv run flask run`
 7. Login using one of the accounts below and you should be good to go!
 
+
+### Database commands. 
+- Create and Seed the Database: `pipenv run manage.py create`
+- Seed the Database: `pipenv run manage.py recreate`
```
Comment (NickSchimek) [inline README.md]: Does recreate seed the database?

**Comment by NickSchimek** [comment]
Why is the user test file being deleted? 🤔

File: README.md (hunk 1 above)
Comment (htharker42) [reply README.md]: I believe so. Let me double check

File: README.md [hunk 2]
```diff
@@ -18,13 +18,16 @@ NOTE: Database is SQLite3 via SQLAlchemy
...
+
 1. Clone the repo (`git clone https://github.com/codeforpdx/dwellinglybackend.git`)
 2. Install Python ( https://realpython.com/installing-python/ )
 3. Install [pipenv](https://pipenv.pypa.io/en/latest/#install-pipenv-today)
 4. Run `pipenv install -d`
-   - If you get the error `ImportError: cannot import name 'Feature' from 'setuptools'`, your setuptools version might be at 46 or later. You may be able to get it to work using version 45 (e.g. `pip3 install setuptools==45`)
+   - If you get the error `ImportError: cannot import name 'Feature' from 'setuptools'`, your setuptools version might be at 46 or later. You may be able to get it to work using version 45 (e.g. `pip3 install setuptools==45`) 
 5. Seed the database
-   - Run: `pipenv run python seed_db.py`
+   - Run: `pipenv run manage.py create`
```
Comment (NickSchimek) [inline README.md]: this should be `pipenv run python manage.py create`
```
5. Create and Seed the database
        Run: `pipenv run python manage.py create`
```

File: README.md (hunk 1 above)
Comment (NickSchimek) [reply README.md]: I've checked on this. `recreate` will drop, create, and seed db.
All manage.py commands need to be prefixed with `pipenv run python`

File: tests/integration/test_users.py [hunk 3]
```diff
@@ -1,151 +0,0 @@
-from models.user import UserModel
-from conftest import is_valid, log
-from freezegun import freeze_time
-from models.user import RoleEnum
-
-def test_user_auth(client, test_database, admin_user):
```
Comment (NickSchimek) [inline tests/integration/test_users.py]: I'm not sure why this file is being removed.  if it is not needed perhaps we can do this in a separate PR.

**Review by NickSchimek — CHANGES_REQUESTED** [review]
@htharker42 the commands as written do not work on my machine. Please check comments for what I think they should be. 
Also, I would prefer not to remove test files in a documentation PR. If these tests must be removed can we create a separate issue/PR for it so that we can document why the removal is needed?

File: README.md (hunk 1 above)
Comment (htharker42) [reply README.md]: you are correct...

**Review by NickSchimek — APPROVED** [review]
Looks good 👍
